magic-links/subscription-management/subscription-management.php

```diff
@@ -169,9 +169,9 @@ public function get_download_url() {
         return $download_url;
     }
 
-    public function echo_calendar_download() {
+    public static function echo_calendar_download( $post_id, $capture_output = false ) {
```

Comment:
@kodinkat can you change this to be a function that return the calendar content. (using `ob_start()`?)
Then it calling context can choose to echo it or send it or whatever.